Terminal.Gui/Views/TextView.cs

```diff
@@ -1995,7 +1995,14 @@ void WrapTextModel ()
 		/// Indicates whatever the text was changed or not.
 		/// <see langword="true"/> if the text was changed <see langword="false"/> otherwise.
 		/// </summary>
-		public bool IsDirty => _historyText.IsDirty (Text);
+		public bool IsDirty {
+			get {
+				return _historyText.IsDirty (Text);
+			}
+			set {
+				_historyText.Clear (Text);
```

Comment:
This change is unrelated to glyphs? Could you give some insight into what it's for?

Not a problem just curious.

Reply:
Well I'm also curious with this. Do you want, if the value is set to false then clear the `_historyText`? If so, there is the `ClearHistoryChanges` method for that.

Reply:
Fixes a bug in config editor where i want isdirty to be cleared after saving. Will use the method BDisp suggested. Didn't see that.